locales-src/asrouter.ftl

```diff
@@ -81,3 +81,29 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Recommended features
+
+cfr-feature-type-recommended = Recommended Feature
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-header = Get this bookmark on your phone
+cfr-sync-bookmarks-body = Take your bookmarks, passwords, history and more everywhere you’re signed into { -brand-product-name }.
+cfr-sync-bookmarks-cta-button = Turn on { -sync-brand-short-name }
+  .accesskey = T
+
+## Send Tab
+
+cfr-send-tab-header-1 = Read this on the go
+cfr-send-tab-header-2 = Take this recipe to the kitchen
```

Comment:
A more descriptive id might be useful to clarify they're different headers not part 1 and part 2 of a related header. Or maybe just `cfr-send-tab-header` vs `cfr-send-tab-recipe-header` ?

Reply:
Second option sounds much clearer, I'll do that! Thanks for all the help on this one :)